src/main/resources/GitHub_True_Dark.theme.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "GitHub True Dark",
```

Comment:
Could you attach few screenshots of theme, Needs to understand if this name is suitable.

Reply:
Attached are the screenshots of the theme in use
![Screenshot_1](https://user-images.githubusercontent.com/46539758/173058864-0d30c28c-fbdf-4420-b29f-a73d611f0166.png)
![Screenshot_2](https://user-images.githubusercontent.com/46539758/173058872-ce735e24-a816-4f13-a215-a7433bd9facf.png)
![Screenshot_3](https://user-images.githubusercontent.com/46539758/173058875-d72213b6-7f1d-46a8-877d-4bf096f8388a.png)

Reply:
@manuelpoelzl isnt it the default dracula theme from jetbrains.. what is the difference?? we can use the same instead of this.

Reply:
It's darker than the default Darcula theme and respects the code color coding of GitHub.